src/api/libcellml/importsource.h

```diff
@@ -109,6 +117,161 @@ class LIBCELLML_EXPORT ImportSource: public Entity
      */
     ImportSourcePtr clone() const;
 
+    /**
+     * @brief Add a component to this import source item.
+     *
+     * The added @p component will be treated as an import, and
+     * will be sourced from the combination of url of this import source
+     * item, and the component's own import reference.
+     *
+     * @param component The @c ComponentPtr to add.
```

Comment:
"Component", not "ComponentPtr".

Reply:
... but the parameter isn't a `Component`, it's a `ComponentPtr`?

Reply:
Same elsewhere...

Reply:
Actually, the documentation is not consistent. See the one for `removeComponent()`.

Reply:
Will be addressed in #618